src/recordlinker/main.py

```diff
@@ -156,19 +193,20 @@ async def link_record(
             updated_bundle=input_bundle,
             message="Supplied bundle contains no Patient resource to link on."
         )
+    
+    #convert record to PII
+    pii_record: schemas.PIIRecord = link.fhir_record_to_pii_record(record_to_link)
 
     # Now link the record
     try:
-        # Make a copy of record_to_link so we don't modify the original
-        record = copy.deepcopy(record_to_link)
-        (found_match, new_person_id) = link.link_record_against_mpi(
-            record=record,
+        (found_match, new_person_id, _patient_reference_id) = link.link_record_against_mpi(
```

Comment:
```suggestion
        (found_match, new_person_id, _) = link.link_record_against_mpi(
```
In python, typically if you have no intentions on using the variable, you just assign it to `_`